addon/components/frost-bunsen-input-select/component.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash'
+import Ember from 'ember'
+import computed, {readOnly} from 'ember-computed-decorators'
+
+import InputMixin from 'ember-frost-bunsen/mixins/input'
+import ListInputMixin from 'ember-frost-bunsen/mixins/list-input'
+import layout from './template'
+
+export default Ember.Component.extend(InputMixin, ListInputMixin, {
```

Comment:
> I no longer have to ask myself what crazy magic is being done…

We are still in the world of Ember so asking yourself what crazy magic is going on may still occur. :trollface: